docker/hip/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG BASE=rocm/dev-ubuntu-20.04:5.2
+FROM $BASE
+
+ARG ADDITIONAL_PACKAGES
+
+RUN apt-get update && apt-get install -y \
+        kmod \
+        wget \
+        ccache \
+        git \
+        file \
```

Comment:
Do you need `file`?

Reply:
I'm not sure, this was in the Kokkos HIP Docker file

Reply:
OK, I got it.